homeassistant/components/control4/config_flow.py

```diff
@@ -150,6 +162,8 @@ async def async_step_init(self, user_input=None):
         if user_input is not None:
             return self.async_create_entry(title="", data=user_input)
 
+        # TODO: figure out how to accept empty strings to disable modes
```

Comment:
I need some help here:
I want users to be able to disable alarm modes that they don't use/aren't supported by their Control4 alarm by leaving the option field blank, but I can't figure out how to get `voluptuous` to accept a blank input as `None`.

This is not as important, but would it be a good idea to hide alarm options if there is no `alarm_control_panel` entity discovered? How would I do that?

Reply:
You can't use `default=` if you want to allow blank input. You can use `description={"suggested_value": <the value>}` instead